### PR TITLE
Launch user docs

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes

What did you change in this pull request? Provide a description of files changed, user interactions changed, etc. Include how to test your changes.

## Related issue

Link to the GitHub issue this pull request addresses using the `#issuenum` format. If it completes an issue, use `Fixes #issuenum` to automatically close the issue.

## Checklist

Things for the developer to confirm they've done before the PR should be accepted:

*   [ ] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
*   [ ] Tests added (or description of how to test) for any new features.
*   [ ] User documentation updated for UI or technical changes.

### CODE REVIEW

File: doc/user/user/pipelines/index.md [hunk 1]
```diff
@@ -60,81 +60,50 @@ Click on the "Select" button at the bottom of the pipeline that you'd like to la
...
+   </blockquote>
+   
+    * If you modify one of the parameters you will be given the option to save the changes for future use.  In the yellow box, just click the "Save template as" button and you will be asked to enter the name for the parameters.  Once saved they can be selected in the dropdown above for future pipelines.
+       ![Save Parameters](images/parameters-modified.png)
+    1. Below these parameters are other settings that can be set (but cannout be saved)
+1. Share Results:
+    1. Some pipeline will allow you to share result back to either the project or sample run.  Checkbox will be visible here if this option is available on this pipeline
+        ![Share Results](images/share-results.png)
+1. Reference Files:
+    1. If the pipeline requires a reference files you will be prompeted to select a reference file from one of the projects or to upload a new one.
```
Comment (deepsidhu85) [inline doc/user/user/pipelines/index.md:90]: Change `If the pipeline requires a reference file, you will be prompted to select a reference file from one of the projects that you have access to, or to upload a new one.`

File: doc/user/user/pipelines/index.md [hunk 2]
```diff
@@ -60,81 +60,50 @@ Click on the "Select" button at the bottom of the pipeline that you'd like to la
...
+       ![Reference file required.](images/reference-file-required.png)
+    2. If you have a **Manager** role on a project, you can [add a reference file to the project](../project/#reference-files). Reference files added to a project can be shared with all other project members.
+    3. If you do not have a **Manager** role on a project, or you would just like to upload a reference file for *this* pipeline, you can add a new reference file to the analysis by clicking on the "Upload Upload" area or dragging a reference file to it. If you upload a reference to the analysis instead of the project, no other users will be able to see or use the file you've uploaded.
+1. Samples are listed last.  Here you can see all the files selected to run.
```
Comment (deepsidhu85) [inline doc/user/user/pipelines/index.md:94]: Change `Here you can see all the files selected to run.` to `Here you can see all the files selected to run the pipeline with.`

File: doc/user/user/pipelines/index.md [hunk 3]
```diff
@@ -60,81 +60,50 @@ Click on the "Select" button at the bottom of the pipeline that you'd like to la
...
-![Rocket ship launch icon.](images/ready-to-launch.png)
-
-Your pipeline submission will be prepared and launched, and you will see a confirmation dialog:
-
-![Pipeline launch success.](images/launch-success.png)
+Once you have made all the modifications to the pipeline and are ready to launch, click the "Launch Pipeline" button at the bottom of the page:
+![Launch Pipeline](images/pipeline-launch.png)
 
-You can [monitor pipeline progress](#monitoring-pipeline-progress) by clicking on the "Let's see how this pipeline is doing" button, or you can empty your cart and start again by clicking on the "Let the pipeline run!" button.
+ONce the pipeline has been launch, you will be directed to the analyses page where you can see the current state of the pipeline, as well as the settings for the pipeline:
```
Comment (deepsidhu85) [inline doc/user/user/pipelines/index.md:105]: Change `ONce the pipeline has been launch, you will be directed to the analyses page where you can see the current state of the pipeline, as well as the settings for the pipeline` to `Once the pipeline has been launched, you will be redirected to the analysis page where you can see the current state of the pipeline, as well as the settings for the pipeline`

File: doc/user/user/pipelines/index.md [hunk 4]
```diff
@@ -60,81 +60,50 @@ Click on the "Select" button at the bottom of the pipeline that you'd like to la
...
 --------------------
 
-Once you've [selected the files](#multiple-filesets-per-sample) and [selected](#using-pre-saved-pipeline-parameters) or [customized](#modifying-pipeline-parameters) the parameters, you can submit the pipeline for execution by clicking on the rocket ship button on the right-hand side of the page:
-
-![Rocket ship launch icon.](images/ready-to-launch.png)
-
-Your pipeline submission will be prepared and launched, and you will see a confirmation dialog:
-
-![Pipeline launch success.](images/launch-success.png)
+Once you have made all the modifications to the pipeline and are ready to launch, click the "Launch Pipeline" button at the bottom of the page:
```
Comment (deepsidhu85) [inline doc/user/user/pipelines/index.md:102]: Change `click the "Launch Pipeline" button at the bottom of the page:` to `select the "Launch Pipeline" button to start the pipeline`

**Review by deepsidhu85 — CHANGES_REQUESTED** [review]
Looking good just a few fixes

File: doc/user/user/pipelines/index.md (hunk 4 above)
Comment (joshsadam) [reply doc/user/user/pipelines/index.md:102]: Updated in f7ca13cdcdb52c5a5c6a287ade42ec432282cadf

File: doc/user/user/pipelines/index.md (hunk 3 above)
Comment (joshsadam) [reply doc/user/user/pipelines/index.md]: Updated in 8ac786e

File: doc/user/user/pipelines/index.md (hunk 2 above)
Comment (joshsadam) [reply doc/user/user/pipelines/index.md]: Updated in b7a8a8d

File: doc/user/user/pipelines/index.md (hunk 1 above)
Comment (joshsadam) [reply doc/user/user/pipelines/index.md]: Updated in 092d339

File: doc/user/user/pipelines/index.md [hunk 5]
```diff
@@ -60,81 +60,50 @@ Click on the "Select" button at the bottom of the pipeline that you'd like to la
...
+    * There are default sets provided by the pipeline developer as well as any that have been saved.
+      ![Default Parameters](images/parameters-default.png)
+    * Parameters can be modified by clicking on the "Modify" button which will expose all available parameters:
+      ![Default Parameters](images/parameters-exposed.png)
+      
+   <blockquote>
+      <p><b>Note</b>: customized pipeline parameters are not validated, please ensure that the values you are entering are valid for the parameter that you are modifying.</p>
+   </blockquote>
+   
+    * If you modify one of the parameters you will be given the option to save the changes for future use.  In the yellow box, just click the "Save template as" button and you will be asked to enter the name for the parameters.  Once saved they can be selected in the dropdown above for future pipelines.
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/user/pipelines/index.md]: Updated in a456f11

File: doc/user/user/pipelines/index.md [hunk 6]
```diff
@@ -60,81 +60,50 @@ Click on the "Select" button at the bottom of the pipeline that you'd like to la
...
-### Using pre-saved pipeline parameters
-
-Parameter sets that [have been saved](#saving-pipeline-parameters) can be used on subsequent executions of the pipeline by using the parameters drop-down, just below the pipeline name:
-
-![Named parameters drop-down.](images/parameters-dropdown.png)
+After selecting a pipeline, you will be sent to the pipeline specific launch page where you can modify the pipeline's general information, parameters, reference file, and have a final look at the files that are goig to be used.
+
+1. General Details:
+   ![General Details](images/general-details.png)
+    * Pipeline Name: A custom pipeline name can be entered allowing you to easily identify the specific analysis at a later data.  By default, the name is given the pipeline name followed by the data run (e.g. SNVPhyl__2-15-2021).
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/user/pipelines/index.md]: Updated in 127b092

File: doc/user/tutorials/snvphyl/index.md [hunk 7]
```diff
@@ -50,27 +50,27 @@ Once the pipeline is selected, the next page provides an overview of all the inp
...
+![snvphyl-pipeline-snv-density.png][]
 
-## Optional parameters
+Checking the checkbox if all the genomes are closely-related to each other and you wish to remove SNVs in high-density regions (that could be indicative of recombination). The thresholds for SNV removal can be set in the [optional parameters][] section.
 
-Next, selecting **Customize** brings up a page where parameters can be customized. The default parameters will often be appropriate, so we will use them for now.
+Leave the checkbox unchecked if you wish to turn off SNV density filtering. This is useful if you are analyzing genomes that are much more distantly related to each other (and so the SNV density filtering would be likely to remove non-recombinant SNVs).
 
-![snvphyl-pipeline-customize][]
+More information on SNV density filtering can be found in the [SNVPhyl documentation][].
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/snvphyl/index.md:74]: Updated in 2558022

File: doc/user/tutorials/snvphyl/index.md [hunk 8]
```diff
@@ -50,27 +50,27 @@ Once the pipeline is selected, the next page provides an overview of all the inp
...
+## Required parameters (density filtering)
 
-Please select **Disable** if you wish to turn off SNV density filtering. This is useful if you are analyzing genomes that are much more distantly related to each other (and so the SNV density filtering would be likely to remove non-recombinant SNVs).
+One component of the SNVPhyl pipeline is to remove regions with high SNV density (which could suggest possible recombination). This component works well when all genomes under question are fairly closely-related, but when analyzing distantly-related genomes the **SNV density filtering** may remove too much data. The **SNV density filtering** can be enabled or disabled using the checkbox provided.
 
-More information on SNV density filtering can be found in the [SNVPhyl documentation][].
+![snvphyl-pipeline-snv-density.png][]
 
-## Optional parameters
+Checking the checkbox if all the genomes are closely-related to each other and you wish to remove SNVs in high-density regions (that could be indicative of recombination). The thresholds for SNV removal can be set in the [optional parameters][] section.
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/snvphyl/index.md]: Updated in 40d8f97

File: doc/user/tutorials/snvphyl/index.md [hunk 9]
```diff
@@ -50,27 +50,27 @@ Once the pipeline is selected, the next page provides an overview of all the inp
...
+SNVPhyl requires a reference genome to be used for mapping sequencing reads and calling variants.  This must be uploaded to the project containing the samples to use.  There is an example reference file in the sample data package (snvphyl-example-lm).  Please upload the file `CFSAN023463.fasta` by clicking the **Click or drag a new reference file to this area to upload.** button.
 
 ![snvphyl-pipeline-upload-ref-file][]
 
-## Required parameters (density filtering)
+## Optional parameters
 
-One component of the SNVPhyl pipeline is to remove regions with high SNV density (which could suggest possible recombination). This component works well when all genomes under question are fairly closely-related, but when analyzing distantly-related genomes the **SNV density filtering** may remove too much data. The **SNV density filtering** can be enabled or disabled using the drop-down menu as shown.
+Next, selecting **Modify** brings up a page where parameters can be customized. The default parameters will often be appropriate, so we will use them for now.
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/snvphyl/index.md]: Updated in 25fb046

File: doc/user/tutorials/snvphyl/index.md [hunk 10]
```diff
@@ -50,27 +50,27 @@ Once the pipeline is selected, the next page provides an overview of all the inp
 
 ## Select a reference genome
 
-SNVPhyl requires a reference genome to be used for mapping sequencing reads and calling variants.  This must be uploaded to the project containing the samples to use.  There is an example reference file in the sample data package (snvphyl-example-lm).  Please upload the file `CFSAN023463.fasta` by clicking the **Upload New** button.
+SNVPhyl requires a reference genome to be used for mapping sequencing reads and calling variants.  This must be uploaded to the project containing the samples to use.  There is an example reference file in the sample data package (snvphyl-example-lm).  Please upload the file `CFSAN023463.fasta` by clicking the **Click or drag a new reference file to this area to upload.** button.
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/snvphyl/index.md]: Updated in f39e26e

File: doc/user/tutorials/sistr/index.md [hunk 11]
```diff
@@ -50,21 +50,17 @@ Please use the **Ready to Launch?** button to start the pipeline.
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/sistr/index.md:50]: Updated in 699da80

File: doc/user/tutorials/sistr/index.md [hunk 12]
```diff
@@ -50,21 +50,17 @@ Please use the **Ready to Launch?** button to start the pipeline.
 
 ![launch-button]
 
-Once the button is selected you should see a screen showing that your pipeline has been launched.
-
-![pipeline-launched]
+Once the button is selected you should be brought to the pipeline detials page.
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/sistr/index.md]: Updated in b5730f5

File: doc/user/tutorials/assembly/index.md [hunk 13]
```diff
@@ -64,22 +64,13 @@ Once the pipeline is selected, the next page provides an overview of all the inp
 
 ![assembly-pipeline-page][]
 
-We will use the default parameters.  Please select the **Ready to Launch?** button to continue.
+We will use the default parameters.  Please select the **Launch Pipeline** button to continue.
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/assembly/index.md]: Updated in c84ec80

File: doc/user/tutorials/refseq_masher/index.md [hunk 14]
```diff
@@ -67,23 +67,13 @@ Modify the parameters to run `refseq_masher` if desired:
 
 ![refseq-pipeline-customize]
 
-Click the "Launch" button to submit the analysis. 
+Click the "Launch Pipeline" button to submit the analysis.
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/refseq_masher/index.md:70]: Updated in fe0a9ea

File: doc/user/tutorials/mentalist/index.md [hunk 15]
```diff
@@ -49,31 +49,16 @@ Once the pipeline is selected, the next page provides an overview of all the inp
...
+Before launching the pipeline, we must select a MentaLiST kmer database to run our samples against. Select an appropriate 'Salmonella enterica' database from the **Kmer DB** from the options displayed. Note that the kmer databases available on your system will have a different date and may have been built with a different value of k than the ones shown below. If there is no 'Salmonella enterica' database available, please contact your IRIDA system administrator and refer to the [administrator documentation][mentalist-admin-docs] for detailed instructions on installing MentaLiST kmer databases.
 
 ![mentalist-pipeline-parameters]
 
-
-Please use the **Ready to Launch?** button to start the pipeline.
+Please use the **Launch Pipeline** button to start the pipeline.
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/mentalist/index.md:56]: Updated in 1146c67

File: doc/user/tutorials/bio_hansel/index.md [hunk 16]
```diff
@@ -54,28 +54,10 @@ You can leave the other parameters unmodified. Please use the **Ready to Launch?
...
-To monitor the status of the launched pipeline, please select the **Analysis > Your Analyses** menu or click the **Let's see how this pipeline is doing** button.
-
-![view-your-analyses]
-
-This will bring you to a page where you can monitor the status of each launched workflow.
-
-![monitor-analyses]
-
-Clicking the pipeline name will bring you to a page for that analysis pipeline. It will take a while for the bio_hansel analysis pipeline to complete. Along the top of the page you can check the current step of the analysis and at the bottom of the **Details** tab you can select if you would like to receive an email upon pipeline completion or error. The email option is only available if the analysis is not in `COMPLETED` or `ERROR` state.
+Once the button is selected you will be redirected to the analysis details page.
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/bio_hansel/index.md]: Updated in a2f122a

File: doc/user/tutorials/mentalist/index.md [hunk 17]
```diff
@@ -49,31 +49,16 @@ Once the pipeline is selected, the next page provides an overview of all the inp
...
-==========================
-
-To monitor the status of the launched pipeline, please select the **Analysis > Your Analyses** menu or click the **Let's see how this pipeline is doing** button.
-
-![view-your-analyses]
-
-The will bring you to a page where you can monitor the status of each launched workflow.
-
-![monitor-analyses]
+Once the button is selected you will be redirected to the analysis details paged.
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/mentalist/index.md]: Updated in 4bac959

[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/mentalist/index.md]: Updated somewhere?...

File: doc/user/tutorials/refseq_masher/index.md [hunk 18]
```diff
@@ -67,23 +67,13 @@ Modify the parameters to run `refseq_masher` if desired:
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/refseq_masher/index.md:67]: Updated somewhere?...

File: doc/user/tutorials/bio_hansel/index.md [hunk 19]
```diff
@@ -54,28 +54,10 @@ You can leave the other parameters unmodified. Please use the **Ready to Launch?
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/bio_hansel/index.md:54]: Updated in ca43bd2

File: doc/user/tutorials/assembly/index.md [hunk 20]
```diff
@@ -64,22 +64,13 @@ Once the pipeline is selected, the next page provides an overview of all the inp
...
 
 ![launch-button][]
 
-Once the button is selected you should see a screen showing that your pipeline has been launched.
+Once the button is selected the pipeline while be initialted and you will be redirected to the analysis details page, where you can monitor the status the workflow.
```
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/assembly/index.md]: Updated in f268182

File: doc/user/tutorials/mentalist/index.md (hunk 17 above)
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/mentalist/index.md]: Updated in 1146c67

File: doc/user/tutorials/refseq_masher/index.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (joshsadam) [reply doc/user/tutorials/refseq_masher/index.md:67]: Updated in fe0a9ea

**Review by deepsidhu85 — APPROVED** [review]

**Pull request merged**